harness/features/allFeatures.local.test.ts

```diff
@@ -32,34 +32,33 @@ describe('allFeatures Tests - Local', () => {
 
         beforeEach(async () => {
             testClient = new LocalTestClient(sdkName)
-            scope
-                .get(
-                    `/client/${testClient.clientId}/config/v1/server/${testClient.sdkKey}.json`,
-                )
-                .reply(200, config)
-
             if (hasCapability(sdkName, Capabilities.sdkConfigEvent)) {
                 scope
                     .post(`/client/${testClient.clientId}/v1/events/batch`)
                     .reply(201, { message: 'Successfully received events.' })
             }
 
+            scope
+                .get(
+                    testClient.getValidConfigPath(),
+                )
+                .reply(200, testClient.getValidConfig)
             await testClient.createClient(true, {
                 configPollingIntervalMS: 60000,
             })
         })
 
-        it('should return all features for user without custom data', async () => {
-            const featuresResponse = await testClient.callAllFeatures({
-                user_id: 'user3',
+            it('should return all features for user without custom data', async () => {
```

Comment:
why is this tabbed in?

Reply:
I have no idea, prettier does what prettier wants